public/css/main.css

```diff
@@ -1244,3 +1244,50 @@ kbd {
     background-color: #fff;
     width: 100%;
 }
+#surveyForm {
+    padding-right:20px;
+}
+#survey-modal-container ol> li {
+    font-weight: bold;
+    font-size: 16px;
+    content: counter(item);
+}
+#survey-modal-container ol> li h4{
+    font-size: 16px;
+    font-weight: bold;
+}
+.survey-question{
+    margin-bottom: 10px;
+}
+.survey-textarea{
+    width: 100%;
+    height: 64px;
+    font-weight: normal;
+    font-size: 14px;
+}
+.liekert{
```

Comment:
likert spelling is incorrect in all of these class names

Reply:
If I remember correctly there was another class already called likert. That was the reason I went along with the existing liekert class name for the survey options.

Reply:
Maybe it was for the survey. I don't see the usage of a likert anywhere else. Why not use that likert class?

Reply:
Regardless, the spelling should be corrected if they were incorrect when you started working on it.

Reply:
Update: I think it might be related to the severity rating scale. But anyway, as I mentioned above, whatever class names are selected, spellings should be corrected.

Reply:
Changed the class names from liekert and liekert-option to survey-option-holder and survey-option